packages/dds/tree/src/simple-tree/treeApi.ts

```diff
@@ -164,6 +172,19 @@ export const treeNodeApi: TreeNodeApi = {
 			T
 		>;
 	},
+	shortID(node: TreeNode): number | undefined {
+		const flexNode = getFlexNode(node);
+		for (const field of flexNode.boxedIterator()) {
+			if (field.schema.kind === FieldKinds.identifier) {
+				for (const child of field.boxedIterator()) {
+					return child.context.nodeKeys.localize(
+						child.value as StableNodeKey,
```

Comment:
does `brand(child.value)` work here instead of the manual cast?

Reply:
Unfortunately `brand(child.value)` doesn't seem to solve this :(